src/main/resources/application.properties

```diff
@@ -23,75 +23,32 @@ cas.send.renew=false
 # =========================================================================
 # Groupings api
 groupings.api.settings=uh-settings
-groupings.api.attribute_assign_id_size=50
-groupings.api.grouping_admins=${groupings.api.settings}:groupingAdmins
-groupings.api.grouping_apps=${groupings.api.settings}:groupingApps
-groupings.api.grouping_owners=${groupings.api.settings}:groupingOwners
-groupings.api.grouping_superusers=${groupings.api.settings}:groupingSuperusers
 groupings.api.attributes=${groupings.api.settings}:attributes
 groupings.api.for_groups=${groupings.api.attributes}:for-groups
-groupings.api.for_memberships=${groupings.api.attributes}:for-memberships
-groupings.api.last_modified=${groupings.api.for_groups}:last-modified
-groupings.api.yyyymmddThhmm=${groupings.api.last_modified}:yyyymmddThhmm
 groupings.api.uhgrouping=${groupings.api.for_groups}:uh-grouping
 groupings.api.destinations=${groupings.api.uhgrouping}:destinations
 groupings.api.listserv=${groupings.api.destinations}:listserv
 groupings.api.ldap=${groupings.api.destinations}:uhReleasedGrouping
-groupings.api.trio=${groupings.api.uhgrouping}:is-trio
-groupings.api.purge_grouping=${groupings.api.uhgrouping}:purge-grouping
-groupings.api.self_opted=${groupings.api.for_memberships}:uh-grouping:self-opted
 groupings.api.anyone_can=${groupings.api.uhgrouping}:anyone-can
 groupings.api.opt_in=${groupings.api.anyone_can}:opt-in
 groupings.api.opt_out=${groupings.api.anyone_can}:opt-out
 groupings.api.basis=:basis
-groupings.api.basis_plus_include=:basis+include
 groupings.api.exclude=:exclude
 groupings.api.include=:include
 groupings.api.owners=:owners
-groupings.api.assign_type_group=group
-groupings.api.assign_type_immediate_membership=imm_mem
-groupings.api.subject_attribute_name_uuid=uid
-groupings.api.operation_assign_attribute=assign_attr
-groupings.api.operation_remove_attribute=remove_attr
-groupings.api.operation_replace_values=replace_values
-groupings.api.privilege_opt_out=optout
-groupings.api.privilege_opt_in=optin
```

Comment:
These two lines are still needed in GroupAttributeServiceImpl.java

Reply:
I believe the isTrio is needed in there as well. Good catch

Reply:
Are you talking about the UI or the API? This pull request is for the UI, not the API. I don't think that file exists on the UI side.

Reply:
> This pull request is for the UI, not the API. I don't think that file exists on the UI side.

oops, sorry 2about that @carolwong49 seems like there was a mix up on the branch that this pr was going into. Well at least now you will know to not remove those from the API haha.